azdev/operations/setup.py

```diff
@@ -83,27 +83,55 @@ def _install_cli(cli_path):
         "install -q -r {}/requirements.txt".format(cli_path),
         "Installing `requirements.txt`..."
     )
+    if deps == 'setup':
```

Comment:
Shall we name the value setup.py to be more specificlly?

```suggestion
    if deps == 'setup.py':
```

Reply:
Also, given the name `deps` is plural, is it designed to support multiple positional parameter values?

Reply:
It means resolve dependencies based on the file specified by the arg value, needs a better name.

Reply:
probably `--deps-in`

Reply:
I think it's based on file type instead file path. The dependecies can only come from two distinct file type, either a set of `setup.py` or a set of `requirements.txt`.

I prefer `--deps-from` , and the valid enum values are _setup.py_ and _requirements.txt_, the default vaule should be one of them. What do you think?

Reply:
Good suggestions.